.github/workflows/release.yml

```diff
@@ -99,6 +105,7 @@ jobs:
   bump-version-generate-changelog:
     name: Bump package version, Generate changelog
     needs: [job-setup]
+    if: ${{ !inputs.only_docker}}
```

Comment:
Minor nit, I think. There should be a space between the variable and the `}}`. I don't think this affects the parsing though.